src/Network/Network/help/New-AzFirewallPolicyNatRule.md

```diff
@@ -139,6 +139,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SourceIpGroup
+The source ipgroups of the rule
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: True
```

Comment:
this should be false, can you fix it manually

Reply:
Fixed